aiographite/graphiteencoder.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python
```

Comment:
how about naming this file to graphite_encode.py? more pythonic and easier to get.